.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow builds and runs all tests
+
+name: Gradle Build and Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  gradle:
+    runs-on: ubuntu-latest
```

Comment:
Is it OK to run on the latest ubuntu? May be better to use a fixed version?

Reply:
fixed